test/CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ set(CMAKE_CXX_STANDARD 20)
 # include(${_project_options_SOURCE_DIR}/Index.cmake)
 include(../src/Index.cmake)
 
+### to get CMAKE_INSTALL_INCLUDEDIR
+include(GNUInstallDirs)
+
```

Comment:
Please move this inside `Index.cmake`.  I already included it inside `PackageProject.cmake`, but I think it is beneficial to load it earlier.

Reply:
I am not sure, if this is than in scope of the CMakeListst.txt?